frontend/src/constants.ts

```diff
@@ -136,15 +140,17 @@ export enum Unit {
 export const INIT_VELOCITY_METRICS: { id: number; name: string; value: string[] }[] = [
   {
     id: 1,
-    name: '2',
-    value: ['dhjejh'],
+    name: 'velocity',
+    value: ['2'],
   },
 ]
 
-export const INIT_CYCLETIME_METRICS: { id: number; name: string; value: string[] }[] = [
+export const INIT_CYCLE_TIME_METRICS: { id: number; name: string; value: string[] }[] = [
   {
     id: 1,
     name: 'Cycle time',
     value: ['40'],
```

Comment:
1. Change this `value` to `values` or `valueList`
2. Change  type of `40` to a `number` not string

Reply:
The data returned from the backend is string.